core/src/impl/Kokkos_ViewMapping.hpp

```diff
@@ -2113,7 +2113,8 @@ struct ViewStride;
 
 template <>
 struct ViewStride<0> {
-  enum { S0 = 0, S1 = 0, S2 = 0, S3 = 0, S4 = 0, S5 = 0, S6 = 0, S7 = 0 };
+  static constexpr size_t S0 = 0, S1 = 0, S2 = 0, S3 = 0, S4 = 0, S5 = 0,
+                          S6 = 0, S7 = 0;
```

Comment:
Uhoh. No please not for now. We had compilers where static constexpr still doesn't work when accessed on the GPU in particular with OpenMP Target

Reply:
I found another workaround.